fbw-common/src/wasm/systems/systems/src/air_conditioning/cabin_pressure_controller.rs

```diff
@@ -222,13 +223,18 @@ impl<C: PressurizationConstants> CabinPressureController<C> {
 
             new_exterior_altitude =
                 self.calculate_altitude(self.exterior_pressure.output(), self.reference_pressure);
-            self.exterior_vertical_speed.update(
-                context.delta(),
-                self.calculate_exterior_vertical_speed(context, new_exterior_altitude),
-            );
+            // When the reference pressure changes, we skip the update to the external
+            // V/S to avoid a jump
+            if self.previous_reference_pressure == self.reference_pressure {
```

Comment:
A tiny note: It feels a bit icky to compare f64s with equality, even though this probably works. What maybe a little nicer is if you use [`f64::EPSILON`](https://doc.rust-lang.org/std/primitive.f64.html#associatedconstant.EPSILON) to check it.

Here's some possible inspiration for how our APU solves it:
https://github.com/flybywiresim/a32nx/blob/8b40cb9a9bca1ed962bdd0000f0e65ccf7e40540/fbw-common/src/wasm/systems/systems/src/apu/air_intake_flap.rs#L58-L59

Reply:
Thanks for the review and the comment @beheh. I've changed it as suggested in two areas where I had direct comparisons of `f64`.